GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/LooseObjectStepTests.cs

```diff
@@ -28,25 +29,64 @@ public LooseObjectStepTests()
         [TestCase]
         public void RemoveLooseObjectsInPackFiles()
         {
-            // Delete any starting loose objects and verify
+            // Delete/Move any starting loose objects and packfiles
             this.DeleteFiles(this.GetLooseObjectFiles());
+            this.MovePackFilesToTemp();
+            Assert.AreEqual(0, this.GetLooseObjectFiles().Count);
+            Assert.AreEqual(0, this.CountPackFiles());

```

Comment:
Create a `Setup` method that does these 4 lines for each test?

Reply:
I actually had to change the test start, so its no longer the same 4 lines